src/login/LoginPage.jsx

```diff
@@ -223,7 +223,14 @@ class LoginPage extends React.Component {
         />
       );
     }
-
+    const tpaAuthenticationError = {};
+    if (thirdPartyAuthContext.errorMessage) {
+      localStorage.removeItem('tpaHintedAuthentication');
```

Comment:
**Context:**
We are setting this in localStorage when the user clicks the Social auth button and the skip_registeration_form is enabled. And when the user completes authentication through social providers and lands back on the register page to complete registration, we show a skeleton (if this value is set) until the MFE context response is received from the API call. (Otherwise it was a UI glitch to show 4 permanent fields and then suddenly hide them on API response)

**Why are we removing it from localStorage here?**

We remove this flag from localStorage when the registration is complete or when there is some error occurred in third-party authentication otherwise the user will end up seeing the spinner on the register page always. This code ensures that what TPA pipeline is failed (the user will have to do a new registration flow), and the user refreshes the register page, he does not see an unlimited spinning skeleton.